### PR TITLE
*: update slow-query doc

### PR DESCRIPTION
**Pull request by crazycs520**
update slow-query doc

### CODE REVIEW

File: sql/slow-query.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ category: user guide
 
 # 慢查询日志
 
-TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1/sql/slow-query.md>)。
+TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1/sql/slow-query.md>)。注：V2.1.8 版本中慢日志里面的字段信息少些，部分字段是 Master 分支后来加入的。
```
Comment (winkyao) [inline sql/slow-query.md]: 注：V2.1.8 版本中慢日志里面的字段信息比较少，部分下面提到的字段是新版本 3.0.0 加入的。

Comment (crazycs520) [reply sql/slow-query.md]: done.

File: sql/slow-query.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ category: user guide
 
 # 慢查询日志
 
-TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1/sql/slow-query.md>)。
+TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1/sql/slow-query.md>)。注：V2.1.8 版本中慢日志里面的字段信息比较少，部分下面提到的字段是新版本 3.0.0 加入的。
```
Comment (lilin90) [inline sql/slow-query.md]:
```suggestion
TiDB 在 V2.1.8 之后更改了慢日志格式，V2.1.8 之前的版本请看[这个文档](<https://github.com/pingcap/docs-cn/blob/master/v2.1/sql/slow-query.md>)。

> **注：**
>
> TiDB 2.1.8 版本中慢日志中的字段信息较少，部分下面提到的字段是在新版本 3.0.0 加入的。
```

File: sql/slow-query.md [hunk 3]
```diff
@@ -16,42 +16,40 @@ TiDB 会将执行时间超过 [slow-threshold](../op-guide/tidb-config-file.md#s
...
 
 ### 字段解析
 
 * `Time`：表示日志打印时间。
 * `Txn_start_ts`：表示事务的开始时间戳，也是事务的 ID，可以用这个值在日志中 grep 出事务相关的日志。
 * `User`：表示执行语句的用户名。
-* `Conn_ID`：表示 connection ID，即 session ID，可以用类似 `con: 3` 的关键字在 TiDB 日志中 grep 出 session ID 为 3 的日志。
+* `Conn_ID`：表示 connection ID，即 session ID, 可以用类似 `con:3` 的关键字在 TiDB 日志中 grep 出 session ID 为 3 的日志。
+* `DB`：表示当前的 database。
+* `Index_ids`：表示语句涉及到的索引的 ID
```
Comment (lilin90) [inline sql/slow-query.md]:
```suggestion
* `Index_ids`：表示语句涉及到的索引的 ID。
```

File: sql/slow-query.md [hunk 4]
```diff
@@ -16,42 +16,40 @@ TiDB 会将执行时间超过 [slow-threshold](../op-guide/tidb-config-file.md#s
...
+* `Is_internal`：表示是否是 TiDB 内部 SQL。true 为TiDB 内部执行的SQL, 比如 analyze，load variable 等；false 为用户执行的 SQL 。
```
Comment (lilin90) [inline sql/slow-query.md]:
```suggestion
* `Is_internal`：表示是否为 TiDB 内部的 SQL 语句。`true` 表示 TiDB 内部执行的 SQL 语句，比如 `analyze`，`load variable` 等；`false` 表示用户执行的 SQL 语句。
```

File: sql/slow-query.md [hunk 5]
```diff
@@ -16,42 +16,40 @@ TiDB 会将执行时间超过 [slow-threshold](../op-guide/tidb-config-file.md#s
...
+* `Digest`：表示 SQL 语句的指纹。
+* `Memory_max`：表示执行期间 TiDB 最多时候使用的内存数量, 单位为byte。
```
Comment (lilin90) [inline sql/slow-query.md]: Could you explain "TiDB 最多时候"? It's hard to understand now. Thanks!

Comment (lilin90) [inline sql/slow-query.md]:
```suggestion
* `Memory_max`：表示执行期间 TiDB 最多时候使用的内存数量，单位为 byte。
```

**Comment by lilin90** [comment]
@crazycs520 Please address the comments.

File: sql/slow-query.md (hunk 5 above)
Comment (qw4990) [reply sql/slow-query.md]: How about "The max bytes of memory TiDB used when processing this query"?

**Comment by huangxiuyan** [comment]
@lilin90 The comments are addressed, PTAL, thanks!

File: sql/slow-query.md (hunk 5 above)
Comment (qw4990) [reply sql/slow-query.md]: @lilin90 PTAL~

**Comment by lilin90** [comment]
@winkyao PTAL again~ Thanks!

**Pull request merged**